docs/connections_and_transactions.md

```diff
@@ -74,7 +74,7 @@ try:
     ...
 except:
     transaction.rollback()
-else:
+finally:
```

Comment:
There is no problem here. See: https://github.com/encode/databases/blob/master/databases/core.py#L328

If you fail the transaction, you never commit it.